cluster/saltbase/salt/kubelet/default

```diff
@@ -41,6 +41,12 @@
 {% endif -%}
 
 {% set config = "--config=/etc/kubernetes/manifests" -%}
+
+{% set manifest_url = "" -%}
+{% if grains['roles'][0] == 'kubernetes-master' and grains.cloud in ['gce'] -%}
```

Comment:
I don't think there's much reason to do it on the nodes, just wanted to think about what it would entail.